hypha/apply/funds/templates/funds/includes/table_filter_and_search.html

```diff
@@ -66,6 +66,11 @@ <h2 class="text-xl mb-2">{{ heading }}</h2>
                     <a href="{% modify_query archived='1' %}">{% trans "Show archived" %}</a>
                 {% endif %}
             {% endif %}
+            {% with request.get_full_path as path %}
+                {% if 'submission' in path and can_export %}
+                    <a href="{{path}}{% if '?' in path %}&{%else%}?{% endif %}export=true"><button type="submit" class="export-btn-width button--primary">{% trans 'Export CSV' %}</button></a>
```

Comment:
Missing a "button" class.

Reply:
Can we style the a-tag as a button directly?

Reply:
Replace `export-btn-width` with `w-max` (tailwind).

Reply:
I couldn't see any difference with `max-w-full` class, but the anchor when styled as `button--primary` looks OK.